conda.recipe/meta.yaml

```diff
@@ -15,31 +15,40 @@ source:
 requirements:
   build:
```

Comment:
We have installation requirements:
* https://github.com/blaze/blaze/blob/master/setup.py#L55-L56
* https://github.com/blaze/blaze/blob/master/requirements-strict.txt

If there's a way to consolidate the `requirements*.txt` files and `meta.yaml`, I'm all ears -- I hate having duplicated stuff.  I haven't found a good way to improve this, however.